### ISSUE TITLE
Add iOS GroupTool view implementation

### ISSUE THREAD

**Issue by maximbircu** [maximbircu/devtools-library#29]
## Abstract
A GroupTool was implemented in #22. It helps the library consumer group a set of dev tools that belongs to the same context. The Android view implementation was already added in #22, we need to and iOS view implementation. 

## Requirements
Add a view implementation for GroupTool.

<img src="https://user-images.githubusercontent.com/12527390/79143832-941ac380-7dc6-11ea-8b66-e56dea0be139.png" width=100>

## Implementation details
The time tool logic is already implemented inside the common module, you just need to provide a view implementation for it.
So, basically you need to:

- Provide an implementation for GroupToolView
- Consume the GroupTooPresenter

## Consumer flow
1. Open the sample app and go to the YML dev tools screen.
1. Find the TimeTool UI control
1. Try to update the configuration value and make sure it's persisted after app restart.

Repeat the same use-case for Memory dev tools screen.
